README.md

```diff
@@ -81,24 +87,61 @@ Wrappers (see [list](https://github.com/gyyang/neurogym/blob/master/docs/wrapper
 are short scripts that allow introducing modifications the original tasks. For instance, the Random Dots Motion task can be transformed into a reaction time task by passing it through the _reaction_time_ wrapper. Alternatively, the _combine_ wrapper allows training an agent in two different tasks simultaneously.
 
 ### Configuration
-`Neurogym` employs a configuration system based on [`Pydantic Settings`](https://docs.pydantic.dev/latest/concepts/pydantic_settings/). You can use a TOML configuration file to create a configuration that can be used throughout the library, for instance, in existing and custom environments (cf. [`neurogym/config/config.py`](neurogym/config/config.py)). The supplied configuration template ([`docs/examples/config.toml`](docs/examples/config.toml)) contains all the currently available options, which mirror the default options in the code.
 
-All options in the configuration code have default values. Rather than changing the values in the code, you can use a TOML file to override the defaults. To do that, just copy the [template configuration](docs/examples/config.toml), change the values of any options that you need to customise, and save it somewhere convenient. You can pass that file to the `Config` class to create a custom configuration vobject:
+🧪 **Beta Feature** — The configuration system is **optional** and currently **under development**. You can still instantiate environments, agents, and wrappers with direct parameters.
+It is only used in a small portion of the codebase and is not required for typical usage.
+See the [`demo.ipynb`](docs/examples/demo.ipynb) notebook for the only current example of this system in action.
+
```

Comment:
Just occurred to me that we could add a note about what such a configuration system would be useful for:

`Using a TOML file can be useful for sharing experiment configurations in a portable way (just send the `config.toml` file to a colleague), reliable saving and loading of experiments and using multiple configurations for the same environment by changing only one line of code.`

Reply:
I've added a comment about this in the docs